WooCommerce/Classes/ViewRelated/Products/Variations/Bulk Update/BulkUpdateViewModel.swift

```diff
@@ -8,6 +8,9 @@ final class BulkUpdateViewModel {
     typealias Section = BulkUpdateViewController.Section
     typealias Row = BulkUpdateViewController.Row
 
+    /// Type that holds the description of an bulk update option, like "Mixed" or "None", and a flag indicating if the value is missing from all variations.
+    typealias UpdateOptionDescription = (text: String, isNotSetForAll: Bool)
```

Comment:
`isNotSetForAll` will be used to display the None description with the more "fade out" color.

Reply:
I created a new VM for the `ValueOneTableViewCell`, not the `BulkUpdateViewModel` provides the `ValueOneTableViewCell.ViewModel` in order for the cell to be configured. Also added unit tests for the new `ValueOneTableViewCell.ViewModel`